### DISCUSSION TITLE
leader key quit working after last update

### DISCUSSION THREAD

**Discussion by odysseyalive** [LazyVim/LazyVim#1120] [General]
I've been running on NVIM v0.9.0 with the same configuration for quite a while now. I haven't had any problems.

After the last update, my leader key no longer works. :(

Any ideas where I would even begin to diagnose the issue at? I've ran checkhealth... nothing is changed in that department.

In my options.lua, I do have the following lines...

```
vim.g.mapleader = ","
vim.g.maplocalleader = ","
```

Even commenting those lines out, doesn't return the leader key to "\\".

what happens now when I run the leader key, is the "jump  to" feature, where it highlights the next letter in the keypress to jump to.

...

Also, I thought Lazyvim had a snapshot system where we could restore it to a previous plugin state.  Am I wrong?

**Comment by folke** [comment]
, is also the default vim binding to go to the previous ftFT result. Flash binds that as well, so that's the issue here.

I'm not home right now, but check the flash docs for `opts.modes.char.keys`. Set that to the default list, but remove the comma so it doesn't get mapped.

**Reply by odysseyalive** [reply]
Thanks for the quick reply. Evidently, I'm not smart enough with this system to apply your suggestion.  What reference do I use in disabled_plugins, in order to prevent flash.nvim from loading. It must be on default, as it is not one of my import options.

**Reply by folke** [reply]
Add the below to a file like `~/.config/nvim/lua/plugins/anything.lua` with the contents:
```lua
return {
    "folke/flash.nvim",
    opts = {
      modes = {
        char = {
          keys = { "f", "F", "t", "T", ";" },
        },
      },
    },
  }

**Reply by odysseyalive** [reply]
Thank you, folke! I see where I made the mistake. This works for me.

**Comment by danielo515** [comment]
Facing the same issue. Will be super awesome if LazyVim stops hijacking my already configured keys. This is the second time in a couple of months where this happens, and it is a waste of time and a big source of frustration
